tests/integration/test_copr_build.py

```diff
@@ -52,10 +52,7 @@ def test_copr_build_existing_project(cwd_upstream_or_distgit, api_instance):
     )
 
     assert build_id == "1"
-    assert url == (
-        "https://copr.fedorainfracloud.org/coprs/"
-        f"{build.ownername}/{build.projectname}/build/{build.id}/"
-    )
+    assert url == (f"https://copr.fedorainfracloud.org/coprs/build/{build.id}/")
```

Comment:
The parentheses are now redundant.

```suggestion
    assert url == f"https://copr.fedorainfracloud.org/coprs/build/{build.id}/"
```